src/release/issue-triaging.md

```diff
@@ -1,125 +1,276 @@
 # Issue triaging
 
-This page is about the `rust-lang/rust` repository. Other repositories may have different processes.
+> This page was last updated on: 2025-01-06.
```

Comment:
Why is this annotation useful? It seems prone to rot, and folks who care can check the git history right?

Reply:
The intention is for a quick remark of when this page was last revised.

Reply:
Dropped